### PR TITLE
Added alt text to the images

### PR DESCRIPTION
**Pull request by lesliealanis**
added descriptive text to the figma images, I'm not sure if its too descriptive and overwhelming so let me know.

### CODE REVIEW

File: content/reports/reports-content.md [hunk 1]
```diff
@@ -23,19 +23,19 @@ Participants can sign up for one of the assigned report cards below, or they can
 
 ## Senate Environment & Public Works Committee:
 
-![](./senate-env-pw.png)
+![A graphic demonstrating the Senate Environment & Public Works Committee: eleven republican members and ten democrat members of the Senate Environment & Public Works Committee. John Barraso is a republican and the chair, Thomas R carper is a democrat and a ranking member](./senate-env-pw.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A graphic demonstrating the Senate Environment & Public Works Committee: eleven republican members and ten democrat members of the Senate Environment & Public Works Committee. John Barraso is a republican and the chair, Thomas R. Carper is a democrat and a ranking member](./senate-env-pw.png)
```

File: content/reports/reports-content.md [hunk 2]
```diff
@@ -23,19 +23,19 @@ Participants can sign up for one of the assigned report cards below, or they can
...
 
-Key:
-![](./key-outlines.png)
-![](./key-progress.png)
+Key
+![A key to the colors used in the graphic: republicans are in red circles and democrats in blue circles, lighter shaded circles indicate the report card is yet to be made, darker shaded circles indicate the reportcard is in progress and circles with a green check indicate the report card is completed] (./key-outlines.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A key to the colors used in the graphic: republicans are in red circles and democrats in blue circles, lighter shaded circles indicate the report card is yet to be made, darker shaded circles indicate the report card is in progress, and circles with a green check indicate the report card is completed] (./key-outlines.png)
```

File: content/reports/reports-content.md [hunk 3]
```diff
@@ -23,19 +23,19 @@ Participants can sign up for one of the assigned report cards below, or they can
...
+![A key to the markings used in the graphic: Committee chairs are circled in black, vice chairs are circled in a dotted black circle and ranking members have four black lines around them.](./key-progress.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A key to the markings used in the graphic: Committee chairs are circled in black, vice chairs are circled in a dotted black circle, and ranking members have four black lines around them.](./key-progress.png)
```

File: content/reports/reports-content.md [hunk 4]
```diff
@@ -23,19 +23,19 @@ Participants can sign up for one of the assigned report cards below, or they can
...
 
 
 ## House Energy & Commerce Committee:
 
-![](house-energy-commerce.png)
+![A graphic demonstrating the House Energy & Commerce Committee: twenty four republican members and thirty one democrat members of the Senate Environment & Public Works Committee. Frank Palone is a democrat and the chair, Yvette D. Clarke is a democrat and the vice chair, Greg Walden is a republican and a ranking member](house-energy-commerce.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A graphic demonstrating the House Energy & Commerce Committee: twenty-four republican members and thirty-one democrat members of the Senate Environment & Public Works Committee. Frank Palone is a democrat and the chair, Yvette D. Clarke is a democrat and the vice chair, Greg Walden is a republican and a ranking member](house-energy-commerce.png)
```

File: content/reports/reports-content.md [hunk 5]
```diff
@@ -23,19 +23,19 @@ Participants can sign up for one of the assigned report cards below, or they can
...
 
 Key:
-![](./key-outlines.png)
+![ to updt](./key-outlines.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]: incomplete?

File: content/reports/reports-content.md [hunk 6]
```diff
@@ -23,19 +23,19 @@ Participants can sign up for one of the assigned report cards below, or they can
...
 ![](./key-progress.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]: this one needs alt text too!

File: content/reports/reports-content.md (hunk 2 above)
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A key to the colors used in the graphic: republicans are in red circles and democrats in blue circles, lighter shaded circles indicate the report card is yet to be made, darker shaded circles indicate the reportcard is in progress and circles with a green check indicate the report card is completed](./key-outlines.png)
```

Comment (Frijol) [reply content/reports/reports-content.md]: Can't add a space here, it will make the image fail to render

**Review by Frijol — CHANGES_REQUESTED** [review]

**Comment by Frijol** [comment]
All of the "Republican" and "Democrat" designations should be capitalized as well

**Comment by lesliealanis** [comment]
Hi @Frijol I made the edits you suggested and commit them directly to the branch I made, I also resolved all the conflicts but I'm not sure where to go from here

File: content/reports/reports-content.md (hunk 1 above)
Comment (Frijol) [reply content/reports/reports-content.md]: It looks like these changes didn't get made, can you check please?

**Review by Frijol — CHANGES_REQUESTED** [review]

File: content/reports/reports-content.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ The [Senate Environment and Public Works Committee](https://www.epw.senate.gov/p
 * Clean Air Act (CAA)
 * Resource Conservation and Recovery Act (RCRA)
 
-![](./report-card-tracker.png)
+![A cycle diagram demonstrating how congress mandates EPA, the EPA issues and enforces permits to facilities that pollute, the ECHO database stores compliance data about these facilities, EEW makes ECHO data easier to understand and EEW works with communities to make environmental compliance report cards which pushes on congress to enforce environmental enforcement through the EPA](./report-card-tracker.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A cycle diagram demonstrating how Congress mandates EPA, the EPA issues and enforces permits to facilities that pollute, the ECHO database stores compliance data about these facilities, EEW makes ECHO data easier to understand and EEW works with communities to make environmental compliance report cards which pushes on congress to enforce environmental enforcement through the EPA](./report-card-tracker.png)
```

File: content/reports/reports-content.md [hunk 8]
```diff
@@ -23,8 +23,8 @@ Participants can sign up for one of the assigned report cards below, or they can
 
 ## Senate Environment & Public Works Committee
 
-![](./senate-env-pw.png)
+![A graphic demonstrating the Senate Environment & Public Works Committee: eleven Republican members and ten Democrat members of the Senate Environment & Public Works Committee. John Barraso is a Republican and the chair, Thomas R Carper is a Democrat and a ranking member](./senate-env-pw.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A graphic demonstrating the Senate Environment & Public Works Committee: eleven Republican members and ten Democrat members of the Senate Environment & Public Works Committee. John Barraso is a Republican and the chair, Thomas R. Carper is a Democrat and a ranking member](./senate-env-pw.png)
```

File: content/reports/reports-content.md [hunk 9]
```diff
@@ -23,8 +23,8 @@ Participants can sign up for one of the assigned report cards below, or they can
...
 
 ## House Energy & Commerce Committee
 
-![](house-energy-commerce.png)
+![A graphic demonstrating the House Energy & Commerce Committee: twenty-four Republican members and thirty-one Democrat members of the Senate Environment & Public Works Committee. Frank Palone is a Democrat and the chair, Yvette D. Clarke is a Democrat and the vice chair, Greg Walden is a Republican and a ranking member](house-energy-commerce.png)
```
Comment (Frijol) [inline content/reports/reports-content.md]:
```suggestion
![A graphic demonstrating the House Energy & Commerce Committee: twenty-four Republican members and thirty-one Democrat members of the Senate Environment & Public Works Committee. Frank Pallone is a Democrat and the chair, Yvette D. Clarke is a Democrat and the vice chair, Greg Walden is a Republican and a ranking member](house-energy-commerce.png)
```

**Review by Frijol — CHANGES_REQUESTED** [review]

**Review by Frijol — APPROVED** [review]

**Pull request merged**